integrations/csm-notification-service/internal/eventbus/config.go

```
@@ -57,3 +62,27 @@ func (c Config) saslMechanism() sasl.Mechanism {
+// consumers permanently idle rather than doing anything wrong. Not used for
+// anything at runtime beyond that one startup check.
+func PartitionCount(ctx context.Context, cfg Config) (int, error) {
+	dialer := &kafka.Dialer{
+		TLS:           &tls.Config{MinVersion: tls.VersionTLS12},
+		SASLMechanism: cfg.saslMechanism(),
+	}
+	conn, err := dialer.DialContext(ctx, "tcp", cfg.Broker)
+	if err != nil {
+		return 0, fmt.Errorf("eventbus: dial %s: %w", cfg.Broker, err)
+	}
+	defer conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)